code-challenges.test.js

```diff
@@ -40,10 +82,38 @@ var arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
 var letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
 
+describe("containVowel", ()=>{
+    test("returns an  array of words that contain a specific letter", ()=>{
+        expect(containVowel(["Apple", "Banana", "Plum", "Orange", "Kiwi"],"a")).toEqual(["Apple", "Banana", "Orange"])
+    })
+    test("returns an  array of words that contain a specific letter", ()=>{
+        expect(containVowel(["Mango", "Cherry", "Apricot", "Blueberry", "Peach"],"e")).toEqual(["Cherry", "Blueberry", "Peach"])
+    })
+})
 
 
 // b) Create the function that makes the test pass.
 
+/*
+    Algorithm->return an array that contain a specific letter
+        Input->an array of words
+        Output->an  array of words that contain a specific letter
+            create a function
+            declare an empty array
+            iterate the given array using for loop
+            convert array[i] to lowercase and use if condition
+            push array[i]that contain the specific letter to the empty array
+            return the created new array
+*/
+
+let containVowel=(array,vowel)=>{
+    let newArray=[]
+    for(let i=0;i<array.length;i++){
+        if(array[i].toLowerCase().includes(vowel)){
+            newArray.push(array[i])
+        }
+    }return newArray
+}
```

Comment:
Well done! I think this code is a good candidate for a refactor using the .filter() method.